packages/components/help-text/src/help-text.css

```diff
@@ -5,11 +5,15 @@
 
 	& dl {
 		color: var(--ts-color-black);
+		margin: 0;
+		padding: 0;
 	}
 
 	& dt,
 	& dd {
+		padding: 0;
 		padding-left: var(--ts-unit);
```

Comment:
May be in one line: `padding: 0 0 0 var(--ts-unit);`

Reply:
I'll do it in the other PR 👍